content/authorization/authorizing-api-calls/index.mdx

```diff
@@ -764,6 +779,126 @@ curl "https://accounts.livechat.com/customer/" \
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `access_token` | A token you can use to call LiveChat APIs on behalf of the user (customer).                                                                                                                                                                                      |
+| `client_id`    | **Client Id** of your app                                                                                                                                                                                                                                        |
+| `entity_id`    | The ID of the newly created customer                                                                                                                                                                                                                             |
+| `expires_in`   | A number in seconds specifying how long the `access_token` will be valid; 28800 sec by default. When it expires, you will need to generate new `access_token` using `refresh_token` (read [Using the refresh token](#using-the-refresh-token) for more details). |
+| `token_type`   | Value: `Bearer`                                                                                                                                                                                                                                                  |
+
+💡 **Got stuck? See common** [**Errors**](#errors).
+
+</Text>
+<Code>
+<CodeSample path={'NEW CUSTOMER'}>
+
+```shell
+curl "https://accounts.livechat.com/customer/token" \
+  -H "Authorization: Bearer <access_token>" \
+  -X POST
+  -d '{
+    "grant_type": "agent_token",
+    "client_id": "9cbf3a968289727cb3cdfe83ab1d9836",
+    "response_type": "token"
+      }'
+```
+
+</CodeSample>
+<CodeResponse title={'Sample response'}>
+
+```json
+{
+  "access_token": "dal:test_SZhSSbShcZxrv580FA",
+  "client_id": "9cbf3a968289727cb3cdfe83ab1d9836",
+  "entity_id": "bf18d1a8-1afe-4a3e-4cc0-a3148f1143db",
+  "expires_in": 28800,
+  "token_type": "Bearer"
+}
+```
+
+</CodeResponse>
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+### Step 2: Existing customer
+
+After creating a new customer identity and using it to perform actions on behalf of the customer, you may want to acquire a new access token for the existing customer - the one you have just created. Make the same request as earlier but this time add an additional parameter: `entity_id`. It can be found in the response in Step 1.
```

Comment:
```suggestion
Use this case to get a new access token for an already existing customer identity. 

Make the same request as in [Case: New customer](#case-new-customer-2), but this time, specify `entity_id` to avoid creating a new identity. You can find your `entity_id` [in the response](#response-5) of the request to create a new customer identity.
```

Reply:
I guess the link should be `#case-new-customer-1`

Reply:
I'm afraid #case-new-customer-1 will link to Cookie grant https://developers.labs.livechat.com/docs/feature/API-8858-customer-sso-docs/authorization/authorizing-api-calls/#step-1-new-customer